cpu/native/syscalls.c

```diff
@@ -145,7 +145,7 @@ void free(void *ptr)
     _native_syscall_leave();
 }
 
-int _native_in_calloc;
+int _native_in_calloc = 0;
```

Comment:
@Kijewski if you think it's bad style I can throw this commit out again, I just thought of it as a form of documentation.
ACK otherwise?